user/deployment/pages.md

```diff
@@ -0,0 +1,48 @@
+---
+title: GitHub Pages Deployment
+layout: en
+permalink: /user/deployment/pages/
+---
+
+Travis CI can deploy your static files to [GitHub
+Pages](https://pages.github.com/) after a successful build.
+
+You will need to provide a [personal access
+token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/)
+and set the deployment provider details in `.travis.yml`.
+
+For a minimal configuration, add the following to your `.travis.yml`:
+
+```yaml
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
+  on:
+    branch: master
+```
+
+> Make sure you have `skip_cleanup` set to true, otherwise Travis CI will delete
+> all the files created during the build, which will probably delete what you are
+> trying to upload.
+
+## Setting the GitHub token
+
+You'll need to generate a [personal access
+token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/)
+with the `public_repo` or `repo` scope. Since the token should be private,
```

Comment:
Sure. I think I went off of the page on [github releases](https://docs.travis-ci.com/user/deployment/releases/) for precedent, but I've just added a note to provide more context. Also, the github.com page where you actually create the token clarifies it a bit as well.